auth-web/tests/unit/views/HomeView.spec.ts

```diff
@@ -0,0 +1,144 @@
+import { createLocalVue, mount } from '@vue/test-utils'
+import BcscPanel from '@/components/auth/BcscPanel.vue'
+import HomeViewDev from '@/views/auth/HomeViewDev.vue'
+import InfoStepper from '@/components/auth/stepper/InfoStepper.vue'
+import LoginBCSC from '@/components/auth/LoginBCSC.vue'
+import TestimonialQuotes from '@/components/auth/TestimonialQuotes.vue'
+import Vue from 'vue'
+import VueRouter from 'vue-router'
+import Vuetify from 'vuetify'
+import Vuex from 'vuex'
+
+Vue.use(Vuetify)
+Vue.use(VueRouter)
+const router = new VueRouter()
+const vuetify = new Vuetify({})
+
+// Prevent the warning "[Vuetify] Unable to locate target [data-app]"
+document.body.setAttribute('data-app', 'true')
+jest.mock('../../../src/plugins/i18n', () => {})
```

Comment:
Would `@` work here to replace "../../../src" ? Wondering if there's a specific requirement for this in the tests.

Reply:
Yes, good point! `@` would certainly work here. 

To be honest, i'm not sure 100% on this mock. I would have expected this mock to way for the mounted component to understand the global methods but it didn't work as how i expected. Let me do a little more research on this mock.

Reply:
So, this wasn't required. I think? the intention was to give definition to such things such as the global methods ( ie $t ), but it didn't quite mock it as expected. Which is why i opted for the approach from above. Removed this in next commit!